samples/bluetooth/mesh/light_ctrl/prj.conf

```diff
@@ -12,6 +12,7 @@ CONFIG_LOG_MODE_DEFERRED=y
 # General configuration
 CONFIG_NCS_APPLICATION_BOOT_BANNER_STRING="Mesh Light Fixture"
 CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+CONFIG_MBEDTLS_HEAP_SIZE=1152
```

Comment:
Please add a comment so that customers can see why this is needed.

Reply:
I don't see the need for an explanation as any other stack etc. does not need an explanation. It is unnecessary IMO

Reply:
Release notes seems enough already;
https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/releases/release-notes-changelog.html

at best I think we can add a release not about PSA change.

Reply:
It should be added to release notes as well since samples' configs are changed in this release. 

However, in future releases this config will stay, but release notes will be different. Adding a note will make clear the need in the config. We need to be more transparent to customers.